README.md

```diff
@@ -283,6 +283,20 @@ Override port (default: `1948`):
 reveal-md slides.md --port 8888
 ```
 
+### Custom Template
+
+Override reveal.js HTML template (default: [this one](https://github.com/webpro/reveal-md/blob/master/lib/template/reveal.html))
+
+``` bash
+reveal-md slides.md --reveal-template my-reveal-template.html
+```
+
+Override listing HTML template (default: [this one](https://github.com/webpro/reveal-md/blob/master/lib/template/reveal.html))
```

Comment:
If you could change the text of the links from "this one" to e.g. "default" or "default template" I'm happy to merge this PR.